dask/bytes/local.py

```diff
@@ -15,51 +15,57 @@
 logger = logging.getLogger(__name__)
 
 
-def read_bytes(fn, delimiter=None, not_zero=False, blocksize=2**27,
-        sample=True, compression=None):
+def read_bytes(path, delimiter=None, not_zero=False, blocksize=2**27,
+        sample=True, compression=None, **kwargs):
```

Comment:
Why do these functions take `kwargs`?  They don't seem to do anything with them.  Is this because we end up gathering bits of information from the path that we send down and no longer need?